src/index.html

```diff
@@ -4,9 +4,225 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,400;1,500;1,700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img
+          src="../src/images/logo.png"
+          alt="logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active"href="#">
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Смартфоны и телефоны
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" data-qa="nav-hover" href="#">Ноутбуки и компьютеры</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              Гаджеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+            Планшеты
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              ФОТО
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="#">
+              ВИДЕО
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
```

Comment:
For readability, add blank lines to separate large or logical code blocks, add them in all places in your code.